doc/getting-started/understanding-config-files.md

```diff
@@ -102,6 +103,17 @@ signal to the `cardano-node` process, e.g. `pkill -HUP cardano-node`. After rece
 signal, `cardano-node` will re-read the file and restart all dns resolution. Please
 **note** that this only applies to the topology configuration file!
 
+If you are synchronizing the network for the first time, please connect to IOG
+relays `relays.cardano-node.iohk.io` by adding it to your local root peers and
+set `useLedgerAfterSlot` to `0` (to disable ledger peers).   You can use
```

Comment:
Where is `useLedgerAfterSlot` being set?

Reply:
That section of the documentation describes the p2p topology file format.  `useLedgerAfterSlot` is part of that.